Proiect Android/app/src/main/java/com/example/myapplication/StartNavigationActivity.java

```diff
@@ -187,7 +181,7 @@ public void getWaypoints(String start, String destination) {
         // for emulator use 10.0.0.2:5000/
         // for device, run ipconfig in cmd and get ipv4 address
 
-        String url = "http://" + getResources().getString(R.string.ip) + ":" + getResources().getString(R.string.port) + "/route/";
```

Comment:
Asta e ok, merita pastrata, dar ca sa iti mearga trebuie sa adaugi in resources->values fisierul neo-settings.xml pus de Dani pe Discord, unde sa iti completezi ip-ul tau